<launch>

    <arg name="camera1_topic" default="/sensor/camera" />

    <node name="camera_publisher" pkg="camera_rospkg" type="camera_publisher" output="screen">
        <!-- <param name="input" type="string" value="$(find camera_rospkg)/recordings/dwecam/arm_camera1.mp4" /> -->
        <param name="input" type="string" value="/dev/video0" />
        <param name="topic_name" type="string" value="$(arg camera1_topic)" />
        <param name="is_wsl2" type="bool" value="false" />
        <param name="is_display" type="bool" value="false" />
        <param name="frame_rate" type="int" value="15" />
        <!-- <param name="whitebalance_temperature" type="int" value="15" /> -->
        <param name="calibration_yaml_path" type="string" value="$(find camera_rospkg)/calibration_yaml/dwe.yaml" />
        <param name="mp4_output_folder" type="string" value="$(find bt_planner)/mp4s/" />
    </node>

</launch>
